### ISSUE TITLE
breadcrumbs : "current" item should not be in tab order

### ISSUE THREAD

**Issue by montoya332** [eBay/ebayui-core#1477]
https://github.com/eBay/ebayui-core/blob/4f33bf56babecc4db9c0483175095a5a3f485afb/src/components/ebay-breadcrumbs/index.marko#L26

https://ebay.github.io/skin/#breadcrumbs
logic results in last element button that is tab-able
`            <a aria-current="location">Smart Watch Bands</a> vs    <button aria-current="location">Smart Watch Bands</button>`

![Screen Shot 2021-07-26 at 6 07 38 PM](https://user-images.githubusercontent.com/8145951/127078979-4b491ac8-513e-43bd-a738-26fdcb7f157a.png)

**Comment by ianmcburnie** [comment]
Seems like an issue for links too: https://ebay.github.io/ebayui-core/?path=/story/ebay-breadcrumbs--last-page-as-current

If breadcrumb has aria-current=location then href should be removed from links, or button set to disabled.

p.s. @montoya332 Buttons for breadcrumbs is a rare use case. In most cases you want your breadcrumbs to be links in order to mirror the deep linkable site hierarchy.

**Comment by ianmcburnie** [comment]
Seems like we have two issues in our storybook. First of all:

1. last-item-parent and last-item-current stories seem like they are the wrong way around. Parent should be a clickable action to the parent item, whereas current should not be clickable at all

Assuming that gets fixed we then have this issue:

2. For last-item-current (now renamed, correctly), a button tag is showing up as the final item in link-based breadcrumbs. We should never mix and match links & buttons in breadcrumbs. This is most likely happening because we fallback to a button when href is not supplied. However, in this case an anchor tag without href attribute should be rendered, not a button.

![Screen Shot 2021-08-24 at 2 49 21 PM](https://user-images.githubusercontent.com/38065/130695479-b1e1f5cd-cfaf-4330-9545-cb1994842e7d.png)

**Comment by ianmcburnie** [comment]
Fixed in 8.0.0 branch.
